packages/crayons-extended/platform-ui/src/styles/_theme.scss

```diff
@@ -0,0 +1,333 @@
+// Color/Theme related config variables
```

Comment:
In the root already there is a folder named styles with most of these style present.

Can you reuse that folder and only add what is unique to this package here ?.

You can refer to how it is done in `crayons-core` package

Reply:
Reused the common CSS